src/mainwindow.cpp

```diff
@@ -2624,11 +2624,13 @@ void MainWindow::mouseMoveEvent(QMouseEvent *event)
 #  endif
 
     if (m_canMoveWindow) {
-        int dx = event->globalX() - m_mousePressX;
-        int dy = event->globalY() - m_mousePressY;
-        move(dx, dy);
-
+        if (!window()->windowHandle()->startSystemMove()) {
```

Comment:
startSystemMove was added in 5.15: https://doc.qt.io/qt-5/qwindow.html#startSystemMove, which is why this fails on the 5.9 build for ubuntu18.04. 

I think you should add a QT_VERSION_CHECK to test if the version is pre 5.15. Wayland users should use 5.15+ anyway.